tests/ChatTests/Groups.hs

```diff
@@ -6610,36 +6914,78 @@ testUpdatedMentionNames = do
       where
         ciMentionMember name = CIMentionMember {groupMemberId = 1, displayName = name, localAlias = Nothing, memberRole = GRMember}
 
-testGroupDirectMessages :: HasCallStack => TestParams -> IO ()
-testGroupDirectMessages =
+testScopedSupportSingleAdmin :: HasCallStack => TestParams -> IO ()
+testScopedSupportSingleAdmin =
   testChat3 aliceProfile bobProfile cathProfile $ \alice bob cath -> do
-    createGroup3 "team" alice bob cath
+    createGroup3' "team" alice (bob, GRMember) (cath, GRMember)
 
     alice #> "#team 1"
     [bob, cath] *<# "#team alice> 1"
 
     bob #> "#team 2"
     [alice, cath] *<# "#team bob> 2"
 
-    void $ withCCTransaction alice $ \db ->
-      DB.execute_ db "UPDATE group_members SET member_status='pending_approval' WHERE group_member_id = 2"
+    alice ##> "/_send #1 @support-2 text 3"
+    alice <# "#team <<support: bob>> 3"
+    bob <# "#team alice> <<support>> 3"
 
-    alice ##> "/_send #1 @2 text 3"
-    alice <# "#team 3"
-    bob <# "#team alice> 3"
+    bob ##> "/_send #1 @support text 4"
+    bob <# "#team <<support>> 4"
+    alice <# "#team bob> <<support: bob>> 4"
 
-    void $ withCCTransaction bob $ \db ->
-      DB.execute_ db "UPDATE group_members SET member_status='pending_approval' WHERE group_member_id = 1"
+    cath ##> "/_send #1 @support-3 text 5"
+    cath <## "#team: you have insufficient permissions for this action, the required role is moderator"
 
-    bob ##> "/_send #1 @1 text 4"
-    bob <# "#team 4"
-    alice <# "#team bob> 4"
+testScopedSupportManyAdmins :: HasCallStack => TestParams -> IO ()
+testScopedSupportManyAdmins =
+  testChat4 aliceProfile bobProfile cathProfile danProfile $ \alice bob cath dan -> do
+    createGroup4 "team" alice (bob, GRMember) (cath, GRMember) (dan, GRModerator)
+
+    threadDelay 1000000
+
+    alice #> "#team 1"
+    [bob, cath, dan] *<# "#team alice> 1"
+
+    threadDelay 1000000
 
-    -- GSMemPendingApproval members don't receive messages sent to group.
-    -- Though in test we got here synthetically, in reality this status
-    -- means they are not yet part of group (not memberCurrent).
-    alice #> "#team 5"
-    cath <# "#team alice> 5"
+    bob #> "#team 2"
+    [alice, cath, dan] *<# "#team bob> 2"
+
+    threadDelay 1000000
+
+    alice ##> "/_send #1 @support-2 text 3"
+    alice <# "#team <<support: bob>> 3"
+    bob <# "#team alice> <<support>> 3"
+    dan <# "#team alice> <<support: bob>> 3"
```

Comment:
```suggestion
    alice <# "#team (support: bob) 3"
    bob <# "#team (support) alice> 3"
    dan <# "#team (support: bob) alice> 3"
```

otherwise it's inconsistent, you are putting message sender in the middle of chat "name"

Reply:
it's quite messy there IIRC with they way from/toGroup are defined.. will see if it's easy fix